### PR TITLE
Alert style fix and plain english page error fix

### PR DESCRIPTION
**Pull request by benjest85**

### CODE REVIEW

**Review by daniel-ac-martin — APPROVED** [review]
Seems fine.

Looks like I need to fix the CI pipeline though!

**Comment by daniel-ac-martin** [comment]
Replaces: #589

File: components/alert/assets/Alert.scss [hunk 1]
```diff
@@ -28,4 +28,8 @@
     @include govuk-font(24, 'bold');
     @include govuk-responsive-margin(3, "bottom");
   }
+
+  a{
```
Comment (daniel-ac-martin) [inline components/alert/assets/Alert.scss:32]: Sorry I just noticed, I think we need to exclude the buttons from this.

Buttons will generally have a class of 'govuk-button'.

Comment (benjest85) [reply components/alert/assets/Alert.scss]: Updated to exclude govuk-button class

**Review by daniel-ac-martin — APPROVED** [review]

**Pull request merged**